Privacy.md

```diff
@@ -0,0 +1,56 @@
+
+- **view network connections**: This is the capability of the app to view information about network connections such as which networks exist and are connected.
+
+- **prevent phone from sleeping**: When running, the app has the capability to prevent your device from sleeping.  
+
+- **run at startup**: This is the ability of the app to startup immediately by itself as soon as the system has finished booting.
+
+<hr>
+
+## Information and data collection
+
+The developer does not externally collect any information or data when the services of BlitzMail is utilized. When performing some activities with the software, personal data is not collected by the developer because all the information are locally stored on your device. 
+### How it works
+Users enter connection information for their email server and this server, of course, knows about the emails that are sent, also their clear-text, but this is normal and necessary. Otherwise, BlitzMail is not making any other internet connection and not sending any data anywhere.
+If an email can't be sent right away, it might be saved locally (on the user's device) until it can be sent.
+
+<hr>
+
+## Crash reports sending
+
+BlitzMail is stable and works fine but you may still experience few errors and crashes, information about these crashes are important to us so the issues can be fixed immediately and the software is running efficiently aga
+If you encounter a bug during the use of this software, please open an [issue](https://github.com/grote/BlitzMail/issues/new) on GitHub and it will be fixed in due time. When submitting a crash report, please provide a logcat to help the developers pinpoint the cause of the crash quickly and fix it accordingly. <hr>
+
+## Service Providers
+
+BlitzMail software does not depend upon the services of third-party software or services to facilitate its utilization and to perform service-related services or to assist in analyzing how the services of the software are used. <hr>
+
+
+## Links To Other Sites
+
+BlitzMail does not contain links to other services. <hr>
+
+## Changes To This Privacy Policy
+
+The developer may update this policy as there will be further development of the software and these developments will likely not be consistent with some statements and terms contained in this current policy.
+It is advisable to check this policy periodically for any changes. Changes to this Privacy Policy are effective when they are made on this document. <hr.
+
+## Disclaimer
+
+The functionality of BlitzMail is designed for general use. The software utilizes not third-party services or tools that may collect your data while you utilize the software. Hence, the developer will not be held accountable for how you decide to use this software and for contents you decide to share via mail and will not be liable for any losses and/or damages in connection with the use of the software. <hr>
```

Comment:
This sounds strange: The software utilizes not third-party services

It *does* utilize the user's mail server though.